src/main/resources/query-selector-shadow-dom.js

```diff
@@ -0,0 +1,337 @@
+(function (selector, element) {
+/// THIRD PARTY LIB CODE STARTS
+/// Sources below transpiled from https://github.com/Georgegriff/query-selector-shadow-dom via npm install && npm run build
```

Comment:
@BorisOsipov  Isn't it outdated comment? The code is not transpiled anymore?

Reply:
it is still transpiled. Added comment with custom rollup bundler config.

Reply:
@asolntsev any quiestions\conserns?

Reply:
No-no, just procrastination:)